src/libs/workflows-app-utils.ts

```diff
@@ -1,13 +1,18 @@
-export const resolveRunningCromwellAppUrl = (apps, currentUser) => {
+import { appStatuses, ListAppResponse } from 'src/libs/ajax/leonardo/models/app-models';
+import { appToolLabels } from 'src/pages/workspaces/workspace/analysis/utils/tool-utils';
+
+export const resolveRunningCromwellAppUrl = (apps: ListAppResponse[], currentUser: string) => {
   // it looks for Kubernetes deployment status RUNNING expressed by Leo
   // See here for specific enumerations -- https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala
   // We explicitly look for a RUNNING app because if the CBAS app is not Running, we won't be able to send import method request.
-  const healthyState = 'RUNNING';
-  const cromwellAppType = 'CROMWELL';
+  const healthyState = appStatuses.running.status;
```

Comment:
@jdcanas 
Is the "correct" approach to this healthy state just to use `'RUNNING'` inline instead of healthyState?

Reply:
Yeah I can move it inline and simply use `RUNNING` instead of `appStatuses.running.status`. I will also move Cromwell app type inline